ushahidi/platform-client
# 762 language settings

This pull request makes the following changes:
- Loads translated languages from a JSON file downloaded from Transifex and falls back to English (US) if the file is not available.

Test these changes by:
- run `gulp transifex-download`
- Log in as admin, go to General settings and check that the languages available are the ones with over 90% translated strings. Currently, 6 languages have over 90% of their strings translated.

Fixes ushahidi/platform#762 .

Ping @ushahidi/platform


[<img src="https://reviewable.io/review_button.svg" height="40" alt="Review on Reviewable"/>](https://reviewable.io/reviews/ushahidi/platform-client/119)
